### ISSUE TITLE
Action executions are suspended

### ISSUE THREAD

**Issue by TonnyL** [actionsflow/actionsflow-workflow-default#1]
<img width="334" alt="Screen Shot 2021-01-24 at 13 49 12" src="https://user-images.githubusercontent.com/13329148/105622326-a6448a80-5e4b-11eb-8819-fece2b0bf7f8.png">

<img width="1116" alt="Screen Shot 2021-01-24 at 13 53 31" src="https://user-images.githubusercontent.com/13329148/105622330-a9d81180-5e4b-11eb-806e-c275120ad3cf.png">

**Comment by hamen** [comment]
I also got an email from github that notified me that I burnt through all my free build minutes for January 🤦🏻‍♂️ A few of the Acrionsflow builds lasted hours before failing 🤦🏻‍♂️ Ouch

**Comment by acidlemon** [comment]
Workaround: I made a little change into my `.github/workflows/actionsflow.yml` to downgrade to old act version.

```patch
       - name: Setup act
         uses: actionsflow/setup-act-for-actionsflow@v1
+        with:
+          version: 0.2.17
```

And, I also set Github Actions timeout to 3min just in case.

```patch
 jobs:
   run:
     runs-on: ubuntu-latest
+    timeout-minutes: 3
```

**Comment by rock3r** [comment]
Looks like the issue is in actionsflow/setup-act-for-actionsflow — when it initializes act, it's not creating the `~/.actrc` file. It should create the file and write something in it, based on one of the three choices (large, medium or micro): https://github.com/nektos/act/blob/56ec36726b384e2f860bbc5b5a9170a3d0c9f511/cmd/root.go#L96-L101

Note that I think the last entry for the `medium` option is missing the `-P` prefix

**Comment by theowenyoung** [comment]
Downgrading the default version to 0.2.18, and I'll fix the 0.2.19 issue~

**Comment by theowenyoung** [comment]
It's ok now, I have fix the act 0.2.19 issues, https://github.com/actionsflow/setup-act-for-actionsflow/issues/1

The v1 has already alias to the latest release, so you don't need do anything.

Feel free for further issues, thanks~
